### PR TITLE
feat post create form added

### PR DESCRIPTION
**Pull request by MustafaTRKYLMZ**

### CODE REVIEW

File: apps/samenvvv/pages/about-us.tsx [hunk 1]
```diff
@@ -1,19 +1,14 @@
 import React from 'react'
 
-import { Box, Heading, Image, SimpleGrid, Stack, Text, VStack } from '@chakra-ui/react' //Stack,
```
Comment (7alip) [inline apps/samenvvv/pages/about-us.tsx:3]: Why do we see these about-us changes which is unrelated to this PR?

File: libs/ui/src/admin/CreateHashtagPostModal/CreateHashtagPostModal.tsx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    reset: resetForm,
+  } = useForm<CreateHashtagPostFormFieldValues>({
+    resolver: yupResolver(schema),
+    mode: 'all',
+  })
+
+  const { mutate, isLoading } = useCreateHashtagPost(locale, queryKey)
+  const toast = useToast()
+  const auth = useAuthSelector()
+  console.log('created', auth.user?.id)
```
Comment (7alip) [inline libs/ui/src/admin/CreateHashtagPostModal/CreateHashtagPostModal.tsx:76]: You can remove logs before pushing your code

File: package.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "@emotion/styled": "^11.10.5",
     "@hookform/resolvers": "^2.9.10",
     "@mollie/api-client": "^3.6.0",
+    "@mui/material": "^5.10.13",
```
Comment (7alip) [inline package.json:30]: For what reason you added `@mui`?

**Review by 7alip — CHANGES_REQUESTED** [review]
Thank you very much; excellent work! I'll look into it again extensively after checking all your works quicky.

File: apps/samenvvv/pages/about-us.tsx (hunk 1 above)
Comment (MustafaTRKYLMZ) [reply apps/samenvvv/pages/about-us.tsx:3]: Because of the lint error. With this error I coudn't push my commit.

File: package.json (hunk 3 above)
Comment (MustafaTRKYLMZ) [reply package.json:30]: I don't remember those changes
